implementation/api-integration.mdx

```diff
@@ -14,4 +14,40 @@ clinical information about each patient as possible (ex: diagnoses, medications)
 This information is shared back with the various health information exchange networks
 as part of their reciprocity requirements.
 
-More information on the Connective Health Clinical Data Processing API can be found [here](/scheduled-appointments/patient-data-processing).
+The Connective Health Clinical Data Processing API specification can be found [here](/scheduled-appointments/patient-data-processing).
+
+## API Implementation Process
+
+### Staging Environment Testing
+
+Before connecting to the production API, integration is tested in the Staging environment.
```

Comment:
Capitalization of production and staging isn't consistent. I assume they should be?

Reply:
yeah, let's keep it lower-case except in titles or beginning of sentences.